docs/content/en/docs/install/install.md

```diff
@@ -100,24 +94,35 @@ Some helpful hints:
 
 ## Modify Helm configuration options
 
-Helm chart values can be modified before the installation.
-This is useful if you want to install only the `metrics-operator`
-rather than the full Toolkit
-or if you need to change the size of the installation.
-
-To modify configuration options, download a copy of the
-[chart/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/values.yaml)
-file, modify some values, and use the modified file to install Keptn:
+To modify the Keptn configuration,
+you must modify the appropriate Helm Chart.
```

Comment:
you never modify helm chart, only values

Reply:
Whoops -- made it `values.yaml` then